ipi-java-220-ex/src/main/java/com/ipiecoles/java/java220/Cadre.java

```diff
@@ -0,0 +1,29 @@
+package com.ipiecoles.java.java220;
+
+import org.joda.time.LocalDate;
+
+import java.util.Objects;
+
+public abstract class Cadre extends Employe{
+    public static Double coefficient = 1d;

```

Comment:
Du coup l'attribut coefficient n'est pas static... et il doit être privé...
```suggestion
    private Double coefficient = 1d;

```